code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -656,6 +656,9 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 	color_suffix = "-y"
 /obj/item/weapon/lighter/green
 	color_suffix = "-g"
+/obj/item/weapon/lighter/NT
```

Comment:
Is NT a reserved word?

Reply:
Doesn't seem like it, or at least doesn't seem like it breaks anything